Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ParameterWindow.cpp

```diff
@@ -251,6 +247,11 @@ namespace EMStudio
     ParameterWindow::~ParameterWindow()
     {
         EMotionFX::AnimGraphNotificationBus::Handler::BusDisconnect();
+
+        if (m_parameterCreateEditWidget)
+        {
+            delete m_parameterCreateEditWidget;
+        }
```

Comment:
update: It's not necessary to check if a pointer is null before calling delete so this can be simplified to just `delete m_parameterCreateEditWidget;`